server/website/website/db/base/parser.py

```diff
@@ -316,6 +320,13 @@ def calculate_change_in_metrics(self, metrics_start, metrics_end):
                     adj_val = end_val - start_val
                 else:  # MetricType.STATISTICS or MetricType.INFO
                     adj_val = end_val
+                if fix_metric_type:
```

Comment:
Among so many Oracle metrics, it is usual that some metrics have wrong types, and break the server.
I propose to fix the metrics type automatically when an error is detected, and warn users by logging.
An issue to be solved is that the previous results are still wrong.